### PR TITLE
Enable auto-update with Sparkle integration

### PR DESCRIPTION
**Pull request by gandarez**
This PR adds the initial Sparkle integration for auto-updating the app.
Screenshot 2023-06-13 at 17 16 56
Screenshot 2023-06-13 at 17 17 22

closes: #21 #66

### CODE REVIEW

**Comment by alanhamlett** [comment]
Can we auto-check the `Automatically download and install updates` checkbox by default?

**Comment by gandarez** [comment]
> Can we auto-check the `Automatically download and install updates` checkbox by default?

According to the [docs](https://sparkle-project.org/documentation/customization/) it's possible. So I set to `YES` the following but I don't know how to make Sparkle ask for auto-update again, do you?
* SUEnableAutomaticChecks
* SUAllowsAutomaticUpdates
* SUAutomaticallyUpdate

File: WakaTime/WakaTimeRelease.entitlements [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (alanhamlett) [inline WakaTime/WakaTimeRelease.entitlements:1]: Is this file used?

**Review by alanhamlett — APPROVED** [review]

File: WakaTime/WakaTimeRelease.entitlements (hunk 1 above)
Comment (gandarez) [reply WakaTime/WakaTimeRelease.entitlements:1]: There's `WakaTime.entitlements` so I believe it was created automatically

Comment (alanhamlett) [reply WakaTime/WakaTimeRelease.entitlements:1]: But we don't have any `WakaTimeRelease.app` so let's try deleting this file?

**Pull request merged**